### ISSUE TITLE
Cannot use two Faker functions with optional data

### ISSUE THREAD

**Issue by jeremy-smith-maco** [nelmio/alice#760]
Something like this fails:

```
Random\Class:
  SomeClass:
    value: '50%? <numberBetween(1, 100)> : <numberBetween(-100, -1)>'
```

Fails with the exception:

`Could not lex the value ">".`

Whereas this passes:

```
Random\Class:
  SomeClass:
    value: '50%? <numberBetween(1, 100)> : ""'
```

Was working in Alice 2.x. Currently upgrading to 3.x.

Current workaround:

```
Random\Class:
  SomeClass:
    value1: '<numberBetween(1, 100)>'
    value2: '<numberBetween(-100, -1)>'
    value: '50%? $value1 : $value2'
```

**Comment by theofidry** [comment]
Same as #761 I'm afraid :/

**Comment by theofidry** [comment]
I'll be closing this one as per the discussion in #761, the real fix is in #601.
